packages/forma-36-react-components/src/components/Flex/Flex.tsx

```diff
@@ -97,44 +89,50 @@ export interface FlexProps {
   /**
    * sets padding-left to one of the corresponding spacing tokens, default is none */
   paddingLeft?: SpacingTypes;
-  /**
-   * style prop, for inline styles */
-  style?: React.CSSProperties;
-}
+};
+
+export type FlexProps<E extends React.ElementType> = PolymorphicComponentProps<
+  E,
+  FlexOwnProps
+>;
```

Comment:
hmmm, I get it now, maybe we use the same pattern as ref `_FlexProps`?

Reply:
not sure, maybe we should state `FlexInternalProps` instead of `FlexOwnProps` to make it clear?

Reply:
updated